### PR TITLE
Apply last example_ynh

### PR DESCRIPTION
**Pull request by yalh76**
## Solution

- *Apply last example_ynh*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by yalh76** [comment]
!testme

File: manifest.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
         "en": "Web interface for Youtube-dl",
         "fr": "Interface Web pour Youtube-dl"
     },
-    "version": "3.0.3~ynh1",
+    "version": "3.0.1~ynh1",
```
Comment (plopoyop) [inline manifest.json]: typo on version ? 
shouldn't it be : `3.0.3~ynh2` ?

Comment (yalh76) [reply manifest.json]: yes, my bad, it was here because of my tests for the autoupgrade script

**Review by plopoyop — APPROVED** [review]
LGTM

**Pull request merged**